Functionality for removing punctuation marks

GoDenisGo opened the pull request:
Extension will now strip the text of any punctuation marks, excluding hyphens.

GoDenisGo committed:
Functionality for removing punctuation marks

Extension will now strip the text of any punctuation marks, excluding hyphens.

JavaRip commented on clickTranslate.js:
```diff
@@ -6,13 +6,46 @@ document.addEventListener('click', async function(event) {
+    let finalString = "";
+
+    for (let i = 0; i < word.length; i++) {
+        const tokenFound = tokensInString(word.charAt(i));
```
the tokens in string function seems redundant to me, you can probably just use `str.replace` with the existing regex

https://stackoverflow.com/questions/6555182/remove-all-special-characters-except-space-from-a-string-using-javascript

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

JavaRip commented:
see if you can refactor out the tokensInString function using `str.replace` I think you might not even need to do any looping

GoDenisGo committed:
Improved trim function on translated words

Words are now trimmed using regexes, for a more concise syntax.

Factored out of the tokensInString() func from previous commits.

Words with numbers 0 to 9 are no longer translated.
